client/my-sites/term-tree-selector/no-results.jsx

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import PureRenderMixin from 'react-pure-render/mixin';
+
+export default React.createClass( {
```

Comment:
What was the reason for moving away from `Component` class to `React.createClass`? Personally I'm strongly in favor of discouraging `React.createClass`.